bin/ember-template-lint.js

```diff
@@ -45,12 +45,13 @@ function lintSource(linter, options, shouldFix) {
 
 function expandFileGlobs(filePatterns, ignorePattern) {
   let result = new Set();
+  let supportedExtensions = new Set(['.hbs', '.html']);
```

Comment:
I think we should technically also support `.handlebars` (though it is super uncommonly used, it is still supported by modern ember-cli apps I think).

Reply:
It's kinda super legacy, do we have issues to add such support?
(land it - no problem, but remove it - may be in future)

Reply:
Ya, maybe? I don't particularly care to be honest, it costs us very little to support which is why I think we should just mirror whatever ember-cli-htmlbars does by default. 

I also think it might be good to allow this exact list to be configurable in the config file.

Reply:
I would suggest to land ability to configure extensions after we land minimal possibility to lint js/ts files, to have stabilyzed logical path to conditionaly apply linting, based on "tricky" extensions

Reply:
@rwjblue one note to land `.handlebars` - we have to rewrite more code, because parts relay on 3symbols ext
![image](https://user-images.githubusercontent.com/1360552/90030138-5ec6a980-dcc4-11ea-810e-39c94c7d4841.png)

Reply:
What parts rely on 3 chars? We should be using `path.extname`...

Reply:
changes landed https://github.com/ember-template-lint/ember-template-lint/pull/1232/commits/768e9c04fe5ffcf9ff04a1f9060d322f4f44dccc